### PR TITLE
adding tests for push notifications apis

### PR DESCRIPTION
**Pull request by ihtram**

### CODE REVIEW

**Comment by ihtram** [comment]
@ziafazal  i have added tests for mobile apps notifications. can you please review?

File: mobileapps/tests.py [hunk 1]
```diff
@@ -573,3 +576,196 @@ def test_mobileapps_organizations_remove(self):
...
         self.assertEqual(response.data['results'][0]['name'], self.organizations[1].name)
+
+
+@ddt.ddt
+class MobileappsNotificationsTests(ModuleStoreTestCase, APIClientMixin):
+    """ Test suite for Mobileapps Notifications API views """
+
+    MODULESTORE = TEST_DATA_SPLIT_MODULESTORE
+
+    def setUp(self):
```
Comment (ziafazal) [inline mobileapps/tests.py]: Can we move the logic in this method into setupClass method instead of setUp?

**Review by ziafazal — CHANGES_REQUESTED** [review]
@ihtram one suggestion other than that looks good.

**Comment by ihtram** [comment]
@ziafazal  i have added setUpClass() 
please review again

**Review by ziafazal — APPROVED** [review]
👍

**Pull request merged**